resources/analysisTools/copyNumberEstimationWorkflow/beagle_create_fake_samples.py

```diff
@@ -0,0 +1,25 @@
+ #!/usr/bin/python
```

Comment:
From the Beagle website:
> Beagle is free software: you can redistribute it and/or modify it under the terms of the GNU General Public License as published by the Free Software Foundation, either version 3 of the License, or (at your option) any later version.

So as far as I understand, we are covered, right?

Reply:
Adding Beagle to the conda environment could also work. I am not sure about the pros and cons for this.

It would work by adding the line `- beagle=5.1_24Aug19.3e8=0` to the `conda.yml` file and then changing the line 
`<tool name="beagle" value="beagle.15Aug19.395.jar" basepath="copyNumberEstimationWorkflow"/>` 
to 
`<tool name="beagle" value="beagle" `
right? (And of course adding it to our conda environment).